doc/REST-interface.md

```diff
@@ -5,6 +5,8 @@ The REST API can be enabled with the `-rest` option.
 
 Supported API
 -------------
+
+####1. Transactions
```

Comment:
I though it will give the reader a better optical identification when a new section starts. But i agree there might be renumbering/rearranging problems in future.
Removed numbers.